src/components/ContentHighlights/ZeroStateCardFooter.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Card } from '@edx/paragon';
+import PropTypes from 'prop-types';
+
+const ZeroStateCardFooter = ({ footerClassNames, children }) => (
+  <Card.Footer className={footerClassNames}>
+    {children}
+  </Card.Footer>
+);
+
+ZeroStateCardFooter.propTypes = {
+  footerClassNames: PropTypes.string,
+  children: PropTypes.node,
+};
+
+ZeroStateCardFooter.defaultProps = {
+  footerClassNames: '',
```

Comment:
nit: default to `undefined` instead to avoid an erroneous `class` attribute without a value